src/index.html

```diff
@@ -1,14 +1,69 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+     name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=2.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <img src="./images/logo.png" alt="MOYO">
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
```

Comment:
use 2 underscores for elements, fix everywhere

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```